candi/bashible/common-steps/all/098_update_node_annotations.sh.tpl

```diff
@@ -0,0 +1,27 @@
+# Copyright 2025 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function kubectl_exec() {
+  kubectl --request-timeout 60s --kubeconfig=/etc/kubernetes/kubelet.conf ${@}
+}
+
+{{- if hasKey .nodeGroup "staticInstances" }}
+if [[ -f /var/lib/bashible/node-spec-provider-id ]]; then
+  PROVIDER_ID="$( kubectl_exec get no "$D8_NODE_HOSTNAME" -o json | jq -r '.spec.providerID' )"
+
```

Comment:
i think we need to check that `/var/lib/bashible/node-spec-provider-id` file is not empty

Reply:
also